src/validations/rules/sap.sch

```diff
@@ -1216,6 +1242,7 @@
             context="oscal:assessment-platform">
             <sch:assert
                 diagnostics="ipv4-has-content-diagnostic"
+                fedramp:specific="true"
```

Comment:
IPv4 addresses are not FedRAMP-specific.
There is an unrelated (to this PR) problem at https://github.com/18F/fedramp-automation/blob/b36d7578885f6b48c1c7650370212e42a124f1e6/src/validations/rules/sap.sch#L1251. The test is not constrained to `oscal:prop[@name eq 'ipv4-address']` – as well, this assertion should have had a context of `oscal:prop[@name eq 'ipv4-address'])`.